lib/ash/policy/authorizer/authorizer.ex

```diff
@@ -775,38 +776,43 @@ defmodule Ash.Policy.Authorizer do
     policies = Ash.Policy.Info.field_policies_for_field(resource, field)
 
     {expr, authorizer} =
-      case strict_check_result(
-             %{
-               authorizer
-               | policies: policies
-             },
-             for_fields: [field],
-             context_description: "accessing field in filter"
-           ) do
-        {:authorized, authorizer} ->
-          {true, authorizer}
-
-        {:error, _} ->
-          {false, authorizer}
-
-        {:filter, authorizer, filter} ->
-          {filter, authorizer}
-
-        {:filter_and_continue, filter, _authorizer} ->
-          raise """
-          Was given a partial filter for a field policy for field #{inspect(field)}.
-
-          Filter: #{inspect(filter)}
-
-          Field policies must currently use only filter checks or simple checks.
-          """
+      if is_nil(policies) do
```

Comment:
I think we should do something like check if the field is a primary key, to be more defensive instead. That way if there is a bug at some point we don't accidentally not apply field policies. Its a better failure case.

Reply:
That sounds good. Let me see if I can figure it out.